signingscript/test/integration/test_autograph.py

```diff
@@ -165,22 +167,36 @@ def _craft_task(file_names, signing_format):
         assert do_verify(signed_path, keyfiles=[mar_pub_key_path]), "Mar signature doesn't match expected key"
 
 
+def _get_java_path(tool_name):
```

Comment:
With `jdk_switcher` this line should be unnecessary, I think. pushapkscript's integration test doesn't need this wrapper: https://github.com/mozilla-releng/pushapkscript/blob/91aaa35f170f6f263b2ebb1dca6c6f44ef6ea8bc/pushapkscript/test/integration/test_integration_script.py#L26